opencraft/settings.py

```diff
@@ -482,6 +482,17 @@
 # Enable or disable celery heartbeats on instances managed by Ocim
 EDX_WORKERS_ENABLE_CELERY_HEARTBEATS = env.bool('EDX_WORKERS_ENABLE_CELERY_HEARTBEATS', default=False)
 
+# Open EdX Instance custom theme for Ocim Managed client instances
```

Comment:
This can be removed if the below suggestion is accepted

Reply:
:+1: